cmd/broker/broker_suite_test.go

```diff
@@ -431,6 +433,22 @@ func (s *BrokerSuiteTest) WaitForLastOperation(iid string, state domain.LastOper
 	return op.ID
 }
 
+func (s *BrokerSuiteTest) WaitForInstanceRemoval(iid string) {
+	err := s.poller.Invoke(func() (done bool, err error) {
+		_, err = s.db.Instances().GetByID(iid)
+		return dberr.IsNotFound(err), nil
+	})
+	assert.NoError(s.t, err, "timeout waiting for the instance %s to be removed", iid)
+}
+
+func (s *BrokerSuiteTest) WaitForBindingRemoval(iid string, bindingID string) {
```

Comment:
No need for that method. Binding removal is not asynchronous.

Reply:
This is exactly needed since removal is asynchronous.

Reply:
Change pushed.